files/zh-cn/web/html/viewport_meta_tag/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: viewport meta 标记
+slug: Web/HTML/Viewport_meta_tag
+---
+
+{{HTMLSidebar}}
+
+本文介绍如何使用"viewport"`<meta>`标记来控制视口的大小和形状。
+
+## 背景
+
+浏览器的{{glossary("viewport", "视口")}}是一个窗口区域，你可以在其中查看 Web 内容。这通常与呈现的页面大小不同，在这种情况下，浏览器为用户提供滚动条以滚动和访问所有内容。
+
+在移动设备和其他窄屏幕中，某些内容在比普通屏幕更宽的虚拟窗口或视口中呈现页面，然后缩小呈现的结果，以便可以一次看到所有内容。然后，用户可以通过平移和缩放以查看页面的不同区域。例如，如果移动屏幕的宽度为 640px，则页面可能会使用 980px 的虚拟视口呈现，然后页面将缩小以适应 640px 的空间。
+
+这是因为并非所有页面都针对移动设备进行了优化，并且在以较小的视口宽度呈现时会被截断（或至少看起来很糟糕）。此虚拟视口是一种使非移动设备优化页面在窄屏设备上看起来更好的一个方法。
+
+但是，此机制不太适用于使用[媒体查询](/zh-cn/docs/Web/CSS/Media_Queries)针对窄屏幕进行优化的页面。例如，如果虚拟视口为 980px，那么在 640px 或 480px 或更低时启动的媒体查询将不会被使用，从而限制了这些响应式设计方法的有效性。 "viewport"`<meta>` 可缓解此类屏幕内部较窄的终端上的虚拟视口问题。
+
+
+## 视口基础知识
+
+典型的针对移动页面优化一般如下所示：
+
+```html
+<meta name="viewport" content="width=device-width, initial-scale=1" />
+```
+
+并非所有的设备都有着相同的宽度，你应该确保即使屏幕大小或方向有很大差异，页面也能够正常工作。
+
+`<meta>` "viewport"的基本属性如下所示：
+
+- `width`
+  - :控制视口的大小。这可以设置为特定像素数（如'width=600'），也可以设置为特殊值`device-width`，即[100vw](/zh-cn/docs/Web/CSS/length#视口 (Viewport) 比例的长度)，即100%的视口宽度。最小值为`1`。最大值为`10000`。负值会被忽略。
+
+- `height`
+
+  - : 控制视口的大小。 这可以设置为特定像素数（如'height=400'），也可以设置为特殊值`device-height`，即[100vh](/zh-cn/docs/Web/CSS/length#视口 (Viewport) 比例的长度)，即100%的视口高度。最小值为`1`。最大值为`10000`。负值会被忽略。
+
+- `initial-scale`
+  - : 控制页面首次加载时显示的缩放倍数。最小值是 `0.1`。最大值是 `10`。默认值为 `1`。负值会被忽略。
+
+- `minimum-scale`
+  - : 控制页面允许缩小的倍数。最小值是 `0.1`。最大值是 `10`。默认值为`1`。负值会被忽略。
+
+- `maximum-scale`
+  - : 控制页面允许放大的倍数。设置一个低于`3`的值将不具备无障碍访问性。最小值是 `0.1`。最大值是 `10`。默认值为`1`。负值会被忽略。
+
+- `user-scalable`
+  - : 控制是否允许页面上的放大和缩小操作。有效值为`0`、`1`、`yes`或`no`。默认值为`1`，与`yes`相同。将值设置为`0`（即与`no`相同）将违反Web内容可访问性指南（WCAG）。
+
+- `interactive-widget`
+  - : 指定交互式UI部件（如虚拟键盘）对页面视口的影响。有效值：`resizes-visual`, `resizes-content`, or `overlays-content`。默认值：`resizes-visual`。
+
+
+> **警告:** 使用`user-scalable=no`会给有视觉障碍的用户造成无障碍问题，如弱视力。[WCAG](/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable#guideline_1.4_make_it_easier_for_users_to_see_and_hear_content_including_separating_foreground_from_background)要求至少有2倍的缩放；然而，最佳做法是启用5倍的缩放。
```

Comment:
https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E7%AB%99%E5%86%85%E9%93%BE%E6%8E%A5%E7%BF%BB%E8%AF%91